app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    files = command.split(" ")
+    for i in range(len(files[2].split("/")[:-1])):
```

Comment:
Use os.makedirs with exist_ok=True to create any missing intermediate directories